### PR TITLE
chore(upgrade): v1.0.0 to polkadot-sdk v1.1.0

### PR DESCRIPTION
**Pull request by enddynayn**
The goal of this PR is to upgrade from v1.0.0 to v1.1.0.

Benchmarks updated for custom pallets and overhead.

References
- [Polkadot Forum](https://forum.polkadot.network/t/polkadot-release-analysis-v1-1-0/3988)
- [Release Notes](https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.1.0)

Closes Issue #1687

### CODE REVIEW

File: runtime/frequency/src/lib.rs
Comment (enddynayn) [inline runtime/frequency/src/lib.rs:797]: revisit for last review

Comment (JoeCap08055) [reply runtime/frequency/src/lib.rs:797]: Some explanation here might be nice. Perhaps I'm reading it backwards, but it looks like these configs would be included for linting OR if we're building with the relay chain enabled... based on these values, I would think we would include them only for an instant-seal node... am I misreading something?

Comment (wilwade) [reply runtime/frequency/src/lib.rs:797]: I believe this is referring to this entire new section about preping for async backing

**Review by wilwade — COMMENTED** [review]
Looks like a straightforward upgrade outside of the Async backing updates.

Few questions, but nothing blocking as far as I can see.

- Reviewed Code
- Ran instant sealing locally

File: runtime/frequency/src/lib.rs
Comment (mattheworris) [reply runtime/frequency/src/lib.rs:797]: These values are for setting up Async Backing. They are the values from the parachain-template.

File: pallets/schemas/Cargo.toml
Comment (wilwade) [reply pallets/schemas/Cargo.toml:30]: Nothing super important like security updates in 1.11.1, so I think it is fine to keep as is then.

File: runtime/frequency/src/lib.rs
Comment (shannonwells) [inline runtime/frequency/src/lib.rs:231]: Non-blocking: Do we have a convention for `use` statements in files and modules?  It seems kind of arbitrary.

**Review by shannonwells — COMMENTED** [review]
I've looked through all these files. Will approve once CI is passing, if needed.

File: runtime/frequency/src/lib.rs
Comment (enddynayn) [reply runtime/frequency/src/lib.rs:231]: This follows templates Polkadot template. We tried to match it as much as possible.

File: pallets/msa/src/weights.rs
Comment (enddynayn) [inline pallets/msa/src/weights.rs:99]: how did we get MaxEncodedLen here?

**Review by wilwade — APPROVED** [review]
Looks good to me.

Might be worth cleaning up the cargo deny warnings, but not a blocker.

- Ran various local tests
- Ran several versions

**Review by aramikm — COMMENTED** [review]
Overall looks good a couple of questions and concerns

- Since try-runtime is separated into a different binary I think out try-runtime script would fail but it was failing before so not a blocker
- concern: Are we sure if there is no internal storage migrations for the frame-pallets that we use?

File: pallets/stateful-storage/src/weights.rs
Comment (wilwade) [reply pallets/stateful-storage/src/weights.rs:77]: It is fine for now. We can get yours in the same release.

File: runtime/frequency/src/lib.rs
Comment (enddynayn) [reply runtime/frequency/src/lib.rs:823]: We can probably do that when we are ready to enable the feature.

File: node/service/src/service.rs
Comment (mattheworris) [reply node/service/src/service.rs:334]: This code comes from the parachain template here: https://github.com/paritytech/polkadot-sdk/blob/495d24d730bb9c2205a5305f383627eabda4a323/cumulus/parachain-template/node/src/service.rs#L291

**Comment by enddynayn** [comment]
> Overall looks good a couple of questions and concerns
> 
> * Since try-runtime is separated into a different binary I think our try-runtime script would fail but it was failing before so not a blocker
> * concern: Are we sure if there is no internal storage migrations for the frame-pallets that we use?

I did not see any migrations but perhaps I missed one @mattheworris?

File: node/service/src/service.rs
Comment (enddynayn) [reply node/service/src/service.rs:334]: https://paritytech.github.io/polkadot-sdk/master/cumulus_client_service/enum.DARecoveryProfile.html

Additionally, it seems these are the only options it takes.

**Comment by mattheworris** [comment]
> > Overall looks good a couple of questions and concerns
> > 
> > * Since try-runtime is separated into a different binary I think our try-runtime script would fail but it was failing before so not a blocker
> > * concern: Are we sure if there is no internal storage migrations for the frame-pallets that we use?
> 
> I did not see any migrations but perhaps I missed one @mattheworris?

There are no migrations called out in the release notes for v1.1.0.

**Review by aramikm — APPROVED** [review]
Great Job! Good to go!

**Review by saraswatpuneet — COMMENTED** [review]
I have reviewed the changes and looks good, however wont be able to approve since I havent run full tests

Otherwise good to go

**Comment by enddynayn** [comment]
> I have reviewed the changes and looks good, however wont be able to approve since I havent run full tests
> 
> Otherwise good to go

Please take the time to run the full test that you need to give it your approval.

File: runtime/common/src/weights/extrinsic_weights.rs
Comment (shannonwells) [inline runtime/common/src/weights/extrinsic_weights.rs:49]: Just noting that this is a jump of ~30% in std-dev which means the weight calculations are varying more widely in this benchmark run.

File: runtime/common/Cargo.toml
Comment (mattheworris) [reply runtime/common/Cargo.toml:15]: It appears that `[build-dependencies]` may not recognize `optional = true` and in testing, removing this dependency has no effects. I am creating a new issue to remove this in a follow-on PR for cleaning up a few minor fixes.

File: node/service/src/service.rs
Comment (JoeCap08055) [reply node/service/src/service.rs:334]: Should we seek more understanding of these values and what they do, or put our faith in the correctness of Parity's template code?

Comment (enddynayn) [reply node/service/src/service.rs:334]: @JoeCap08055 Please double check this but this seems correct to me. Full nodes simply follow the chain they do no create blocks. If you are a validator you should be an authoring node. This leaves us with the collator option.

**Review by shannonwells — APPROVED** [review]
Good work, ship it!

**Review by JoeCap08055 — APPROVED** [review]

**Pull request merged**
